packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -60,20 +71,35 @@ function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLEl
     isRequired,
     label: null,
     isQuiet: false,
-    'aria-labelledby': labelProps.id || ariaLabel,
-    validationState: null
+    validationState: null,
+    description: null,
+    errorMessage: null,
+    descriptionProps: null,
+    errorMessageProps: null
   };
+
   let searchFieldClassName = classNames(styles, 'spectrum-SearchWithin-searchfield');
   let pickerClassName = classNames(styles, 'spectrum-SearchWithin-picker');
+
   let slots = {
-    searchfield: {UNSAFE_className: searchFieldClassName, ...fieldProps, ...defaultSlotValues},
-    picker: {UNSAFE_className: pickerClassName, menuWidth, align: 'end', ...defaultSlotValues}
+    searchfield: {
+      ...defaultSlotValues,
+      ...fieldProps,
+      UNSAFE_className: searchFieldClassName,
+      'aria-labelledby': labelledBy,
+      'aria-label': labels['aria-label']
+    },
```

Comment:
I'm digging right now as to why but it seems like the searchfield has the same id as the group id and ends up self referencing itself via `aria-labelledby`: 
![image](https://user-images.githubusercontent.com/9661127/140991962-f0f81291-de7d-44a1-b00e-6c0a1517ede6.png)
Note that the id of the `group` is the same as the searchfield div in the screenshot.
Correct me if I'm wrong but I believe the id of the searchfield should be a unique value here. Making it a different value from the group id will then have VoiceOver properly announce the label + aria label of the searchfield

Reply:
Fixed by latest commits